### PR TITLE
CAMS-441 case detail critical accessibility

### PR DESCRIPTION
**Pull request by fmaddenflx**
# Purpose

Address Case Details Critical Accessibility Issues

# Major Changes

See ticket notes.  All accessibility issues solved with the exception of a few Date Picker issues that can not be resolved using the native date picker.

# Testing/Validation

All tests are passing and cover over 90% of the code.

# Definition of Done:

- [ ] Acceptance criteria met - The system meets all acceptance criteria
- [x] Feature toggles created - Features that are deployed but not released have toggles
- [ ] Design QA passed - All visual UI looks and works the way it’s supposed to
- [ ] Usability validated - Team ensures the working code is easy to use
- [x] Accessibility tests passed - System passes automated and manual accessibility tests and supports applicable devices
- [x] Code refactored for clarity - Developers can understand the work simply by reviewing the code
- [x] Dependency Rule followed - More important code doesn’t directly depend on less important code
- [x] Development debt eliminated - UX and code aligns to the team’s latest understanding of the domain
- [ ] Source code merged - Code has been merged into the main branch
- [x] 90% unit test coverage achieved - Automated test coverage tools indicate coverage of >= 90%
- [x] Code reviewed - Code is reviewed by at least two other team members before being merged
- [x] Code quality checks passed - Code passes all automated quality checks
- [x] Security scans passed - Code passes expected scans for vulnerabilities and compliance issues
- [x] Threat model updated - The threat model incorporates new threats and remediations
- [x] Build process updated - Automated build process include code supporting the user story
- [x] Load/performance tests passed - Performance tests include the functionality of the user story
- [x] Documents are updated - All required documentation is up to date and version controlled
- [ ] Code is deployed - Code is deployed to the highest production like environment

### CODE REVIEW

File: user-interface/src/lib/components/uswds/DateRangePicker.tsx [hunk 1]
```diff
@@ -103,11 +133,46 @@ function DateRangePickerComponent(props: DateRangePickerProps, ref: React.Ref<Da
...
         label={endDateLabel || 'End date'}
+        ariaDescription={`${id}-aria-description`}
+        ariaLive={ariaLive ?? undefined}
         name="event-date-end"
         value={value?.end}
         disabled={disabled}
         required={required}
       />
+      <span id={`${id}-aria-description`} aria-live="polite" hidden>
+        <span>Format: numeric month / numeric day / 4 digit year.</span>
```
Comment (jamesobrooks) [inline user-interface/src/lib/components/uswds/DateRangePicker.tsx:144]: How does the screen reader read this? 

Also, I believe the following is generally more grammatically correct:
```suggestion
        <span>Format: numeric month / numeric day / 4-digit year.</span>
```

Comment (fmaddenflx) [reply user-interface/src/lib/components/uswds/DateRangePicker.tsx:144]: It reads it as "Format" "numeric month slash numeric day slash 4 digit year"

I'll add the hyphen

File: user-interface/src/case-detail/CaseDetailScreen.tsx [hunk 2]
```diff
@@ -446,15 +456,19 @@ export default function CaseDetailScreen(props: CaseDetailProps) {
                         onPillSelection={handleSelectedFacet}
                         onUpdateSelection={handleFacetClear}
                         label="Filter by Summary"
+                        ariaDescription="Select multiple options. Results will update when the dropdown is closed."
+                        ariaLive="off"
```
Comment (amorrow-flexion) [inline user-interface/src/case-detail/CaseDetailScreen.tsx:460]: for consistency purposes, we should probably make the `ariaDescription & ariaLive` here `aria-description & aria-live` like we have on the DateRangPicker

**Review by amorrow-flexion — APPROVED** [review]

File: user-interface/src/lib/components/uswds/DateRangePicker.tsx (hunk 1 above)
Comment (jamesobrooks) [reply user-interface/src/lib/components/uswds/DateRangePicker.tsx]: Cool. I was hoping it wasn't doing something weird with the slash.

**Review by jamesobrooks — APPROVED** [review]

**Pull request merged**